modules/hal_gigadevice/Kconfig

```diff
@@ -69,6 +69,11 @@ config GD32_LOW_SPEED_IRC_FREQUENCY
 	help
 	  Define value of low speed internal RC oscillator (IRC) in Hz
 
+config GD32_DBG_SUPPORT
+	bool "Use GD32 Debug features"
+	select USE_GD32_DBG
+	default y if DEBUG
+
```

Comment:
I was thinking more about this option. You can switch to:

```diff
-default y if DEBUG
+default y
```

Reply:
OK. Applied it.